mappings/net/minecraft/entity/Living.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/sd net/minecraft/entity/Living
```

Comment:
Yeah, all entity classes should be prefixed with `Entity`, just like items, blocks and such.